templates/nginx/nginx-dep.yaml

```
@@ -86,7 +86,7 @@ spec:
             {{- toYaml .Values.nginx.env | nindent 12 }}
           {{- end }}
         {{- if .Values.nginx.extraContainers }}
-        {{ toYaml .Values.nginx.extraContainers | indent 8}}
+        {{ toYaml .Values.nginx.extraContainers | nindent 8}}
```

`nindent` will add a line break, so you'll want to trim it with `-`
```suggestion
        {{- toYaml .Values.nginx.extraContainers | nindent 8}}
```